core/asm/asm.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014, 2017 The go-ethereum Authors
```

Comment:
Only one year is needed. We usually put the year in which the file was first introduced.